### PR TITLE
updated morae demo notebook

### PR DESCRIPTION
**Pull request by Free-Quarks**
Updating the morae-demo notebook to match the one that was presented on 6/28/23.

### CODE REVIEW

File: morae-demo/.DS_Store
Comment (cl4yton) [inline morae-demo/.DS_Store:1]: Please remove this file -- don't include it in the PR.
(I suspect you might want to include a .gitignore)

**Review by cl4yton — CHANGES_REQUESTED** [review]
@Free-Quarks Everything generally looks good, my only request is that you remove the .DS_Store (a Mac OS-generated file) that is currently included in the PR.

File: morae-demo/.DS_Store
Comment (Free-Quarks) [reply morae-demo/.DS_Store:1]: Just removed them! Thanks!

**Review by cl4yton — APPROVED** [review]
Thanks!

**Pull request merged**